### PR TITLE
Four more holy weapons

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
rscadd: Four more chaplain weapon options have been added: hanzo steel, light energy sword, dark energy sword, and monk's staff.
/:cl:

Hanzo steel, red energy sword, blue energy sword, monk's staff.

All of them have the same strength/melee only block as the claymore, save for the monk staff which has slightly higher block and slightly less strength.

### CODE REVIEW

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 1]
```diff
@@ -118,6 +118,33 @@
...
+	icon_state = "katana"
+	item_state = "katana"
+	slot_flags = SLOT_BELT | SLOT_BACK
+
+/obj/item/weapon/nullrod/claymore/saber
+	name = "light energy sword"
+	hitsound = 'sound/weapons/blade1.ogg'
+	icon_state = "swordblue"
+	item_state = "swordblue"
+	desc = "If you strike me down, I shall adminhelp more obnoxiously than you can possibly imagine."
```
Comment (Shadowlight213) [inline code/game/objects/items/weapons/holy_weapons.dm]: adminhelp is too OOC for an IC description

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 2]
```diff
@@ -118,6 +118,33 @@
...
+	name = "dark energy sword"
+	icon_state = "swordred"
+	item_state = "swordred"
+	desc = "Woefully ineffective when used on steep terrain."
+
+/obj/item/weapon/nullrod/claymore/saber/red/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance, damage, attack_type)
+	if(prob(20))
+		var/phrase = pick("No...no no, YOU will die!!!", "YEAAAAAAAARGH!!!", "I'm too weak, please, don't kill me!!!", "It's treason, then!")
+		owner.say("[phrase]")
+	..()
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/holy_weapons.dm]: @hornygranny

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/objects/items/weapons/holy_weapons.dm]: Need an alternate joke then. I coulda sworn we had other 4th wall breaking descriptions though

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 2 above)
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/holy_weapons.dm]: Just...no. This is retarded.

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 3]
```diff
@@ -227,4 +254,18 @@
...
\ No newline at end of file
+	used_blessing = TRUE
+
+/obj/item/weapon/nullrod/claymore/bostaff //May as well make it a "claymore" and inherit the blocking
+	name = "monk's staff"
+	desc = "A long, tall staff made of polished wood. Traditionally used in ancient old-Earth martial arts, now used to harass the clown."
+	w_class = 4
+	force = 15
+	block_chance = 40 //Doesn't fit on your back, so a bit better at blocking
+	slot_flags = SLOT_BACK
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/holy_weapons.dm]: It doesn't fit on your back, but pstt shhh, it fits on your back.

**Comment by Fox-McCloud** [comment]
The Monk Staff is completely superior to the regular staves.

The regular staves have a force of 5, w class of 5, and only fit on your back...with a block chance of 50.

This has a block chance of 40, with the same weaknesses, but have +10 extra force.


Why would someone ever want to pick the staff over this?

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/holy_weapons.dm]: "If you strike me down, I shall become more robust than you can possibly imagine."

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 3 above)
Comment (PKPenguin321) [inline code/game/objects/items/weapons/holy_weapons.dm]: probably a bug fam calm down

Comment (KorPhaeron) [inline code/game/objects/items/weapons/holy_weapons.dm]: Err, not on belt I meant

**Comment by KorPhaeron** [comment]
@Fox-McCloud 

Monk staff only blocks melee, the god staves block bullets

File: code/game/objects/items/weapons/holy_weapons.dm (hunk 2 above)
Comment (KorPhaeron) [inline code/game/objects/items/weapons/holy_weapons.dm]: That is true. I am open to removing it

**Comment by The-Albinobigfoot** [comment]
I believe we're beginning to fly far too closely to the sun.

**Comment by Fox-McCloud** [comment]
@KorPhaeron 

I see, thanks for the clarification.

**Comment by KazeEspada** [comment]
Monk staff uses which staff model?

**Comment by KorPhaeron** [comment]
bo staff

**Pull request merged**